cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -1454,7 +1454,7 @@ func TestDeployedDNSAddon(t *testing.T) {
 		},
 		{
 			name:           "with digest",
-			image:          "registry.k8s.io/coredns/coredns:v1.12.3@sha256:1391544c978029fcddc65068f6ad67f396e55585b664ecccd7fefba029b9b706",
+			image:          "registry.k8s.io/coredns/coredns:v1.12.4@sha256:986f04c2e15e147d00bdd51e8c51bcef3644b13ff806be7d2ff1b261d6dfbae1",
```

Comment:
does this test pass if the image version and sha are  not matching the version in constants.go?
if yes, please change this to some arbitrary tag/sha so that we don't have to update it every time coredns bumps, or don't change it in this PR.